src/server/environment.js

```diff
@@ -1,27 +1,17 @@
 
-const { tmpdir } = require('os')
-const { mkdtempSync } = require('fs')
 const { join, resolve } = require('path')
 
-function tempDataDir () {
-  return mkdtempSync(join(tmpdir(), 'arpa-data-'))
-}
-
 const VERBOSE = Boolean(process.env.VERBOSE)
 
-const DATA_DIR = resolve(process.env.DATA_DIR || tempDataDir())
```

Comment:
yeah, good question, why is this like that? fully support removing the temp functionality here.